lib/diagnostics_channel.js

```diff
@@ -250,33 +251,40 @@ function assertChannel(value, name) {
   }
 }
 
+function channelOrName(nameOrChannels, name) {
```

Comment:
Can you add a jsdoc for these parameters and function?

Reply:
It's not public-facing though. Are we doing jsdoc on private functions? 🤔

Reply:
It helps with intellisense and developer experience. There is no enforcement, so it is entirely up to you :-)